### ISSUE TITLE
9_Combined_AspId_and_EFStorage - Build Error

### ISSUE THREAD

**Issue by valentinacupac** [IdentityServer/IdentityServer4#3419]
Please only use the issue tracker for bug reports and/or feature requests. For general security questions, or free or commercial support options do __not__ use the issue tracker and instead see [here](http://docs.identityserver.io/en/latest/intro/support.html) for more details.

For bug reports,  include the relevant log files related to your issue. See here how to enable [logging](https://identityserver4.readthedocs.io/en/release/topics/logging.html). Delete this line once you have.

Finally, please keep the issue concise and to the point. If you paste in more code than the text for the issue you are reporting then we will most likely not read it. 

### Issue / Steps to reproduce the problem

1. Cloned the repository
2. Opened IdentityServer4\samples\Quickstarts\9_Combined_AspId_and_EFStorage\Combined.sln in Visual Studio 2019.
3. Re-built the solution. Got build errors.
4. Tried also NuGet package restore. Got build errors.

See image:
![image](https://user-images.githubusercontent.com/677159/61070415-f0535000-a40e-11e9-87b1-9bdee0c16c55.png)

### Relevant parts of the log file

```
<log goes here>
```

**Comment by brockallen** [comment]
perhaps a clone issue. it builds for me.

**Comment by valentinacupac** [comment]
I tried cloning again, and got build issue again, so I tried to see what could be the possible causes.

I also tried to open up another solution, IdentityServer4.Core.sln, and I get the same error as for the sample project, so it seems I cannot build any solution from IdentityServer4...

When trying to do nuget restore: Error occurred while restoring NuGet packages: The local source 'D:\GitHub\IdentityServer\IdentityServer4\nuget' doesn't exist.			

![image](https://user-images.githubusercontent.com/677159/61074247-90ad7280-a417-11e9-8d8b-03fc2b6be8c6.png)

![image](https://user-images.githubusercontent.com/677159/61074263-999e4400-a417-11e9-86a8-f426694691b5.png)

![image](https://user-images.githubusercontent.com/677159/61074232-82f7ed00-a417-11e9-9b75-1859c52a8dfa.png)

**Comment by valentinacupac** [comment]
UPDATE: The issue was caused by NuGet localsource, I found the solution here: https://stackoverflow.com/questions/49446394/unable-to-find-nuget-local-source-error-occurred-while-restoring-nuget-packages

So I did the following, in case it occurs to someone else in the future:
1. Tools > Options > NuGet Package Manager
2. I saw that inside Available sources "IdentityServer Local" was referencing <PathOnMyFileSystem>\IdentityServer4\./nuget
3. So I changed it to <PathOnMyFileSystem>\IdentityServer4

Now the nuget packages are able to be restored and now the build works.
